.github/workflows/run-tests-dev.yml

```diff
@@ -34,6 +34,7 @@ jobs:
                  Multiple_Condition_Runs,
                  Nonhydro_Body,
                  Nonlinear_Hydro,
+                 Nonlinear_Model_Predictive_Controller_for_WECCCOMP,
```

Comment:
This doesn't match the folder name (Nonlinear_Model_Predictive_for_WECCCOMP)

Reply:
Fixed